kumascript/tests/macros/apiref.test.js

```diff
@@ -349,16 +349,16 @@ function checkInterfaceItem(actual, expected, config) {
     // and the text contents omits the CTA
     const methodLink = actual.querySelector("a");
     expect(methodLink).toBeNull();
-    const methodName = actual.querySelector("svg");
+    const methodName = actual.querySelector("abbr, svg");
```

Comment:
Would `.icon` work as well (more descriptive)?

```suggestion
    const methodName = actual.querySelector(".icon");
```